packages/api-reference/src/components/Content/Content.vue

```diff
@@ -90,8 +90,18 @@ const introCardsSlot = computed(() =>
       :server="activeServer" />
 
     <Introduction
-      v-if="document?.info?.title || document?.info?.description"
-      :document="document">
+      v-if="
+        document?.info?.title ||
+        document?.info?.description ||
+        document?.info?.contact ||
+        document?.info?.license ||
+        document?.info?.termsOfService
+      "
```

Comment:
does it actually look good if only one of those is provided? if not, let’s keep the old v-if